oe_theme.theme

```diff
@@ -932,6 +933,15 @@ function oe_theme_preprocess_pattern_date_block__preview(array &$variables): voi
   $variables['date'] = DateValueObject::fromArray($variables['date']);
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_pattern_text_featured_media__preview(array &$variables): void {
+  if (is_array($variables['image'])) {
```

Comment:
No need for this if, we know that's an array since it's in the pattern preview definition, this is the `__preview` preprocess suggestion after all.

Reply:
@ademarco the problem is that you have asked for removing the image from the test completely. Without this it will fail.